lib/unlocked/EditTile.dart

```diff
@@ -0,0 +1,485 @@
+                    height: 168,
+                    child: ClipRRect(
+                      borderRadius: BorderRadius.circular(8.0),
+                      child: Card(
+                        child: Column(
+                          mainAxisAlignment: MainAxisAlignment.center,
+                          crossAxisAlignment: CrossAxisAlignment.center,
+                          children: <Widget>[
+                            Expanded(
+                              child: Container(
+                                color: Color(0xffF8C199),
+                                child: GestureDetector(
+                                  onTap: () => {
+                                    showModalBottomSheet(
+                                      context: context,
+                                      builder: (context) {
+                                        return Wrap(
+                                          children: [
+                                            ListTile(
+                                              leading: Icon(
+                                                Icons.photo_camera,
+                                                color: Color(0xff23036A),
+                                              ),
+                                              title: Text('Take Photos'),
+                                            ),
+                                            ListTile(
+                                              leading: Icon(
+                                                Icons.insert_photo,
+                                                color: Color(0xff23036A),
+                                              ),
+                                              title: Text('Browse Album'),
+                                            ),
+                                            ListTile(
+                                              leading: Icon(
+                                                Icons.search,
+                                                color: Color(0xff23036A),
+                                              ),
+                                              title: Text('Search Community'),
+                                            ),
+                                          ],
+                                        );
+                                      },
+                                    )
+                                  },
+                                  child: Image.asset(
+                                    widget.tile.imageUrl,
+                                    width: screenSize.width / 2,
+                                  ),
+                                ),
+                              ),
+                            ),
+                            Container(
+                              height: 24,
+                              child: Text(widget.tile.name,
+                                  style: TextStyle(
```

Comment:
This looks like it'd make a cool `Typography` style - pull from `master` to see how it comes together.